src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -25,6 +25,42 @@ class UserInfoSection extends React.Component {
       );
     }
 
+    let entitlementInfo;
+    if (enrollmentData.percentageBenefit === 0 && enrollmentData.originalEntitlement === 0) {
+      entitlementInfo = (
+        <div>
+          <h4>When You Can Receive Benefits</h4>
+          <div className="usa-alert usa-alert-warning usa-content">
+            <div className="usa-alert-body">
+              <h2>Currently Disallowed</h2>
+              We have received your application and have determined that you are not currently eligible
+              for Post-9/11 GI Bill benefits. Additional service time could change this determination.
+            </div>
+          </div>
+        </div>
+      );
+    } else {
+      entitlementInfo = (
+        <div>
+          <div>
+            <h4>When You Can Receive Benefits</h4>
```

Comment:
I think it would be a little more readable to split these into 2 variables, maybe `entitlementInfo` for the stuff under  "When You Can Receive Benefits" and `benefitLevel` for the stuff under "Your Benefit Level".

Then the logic can be something like
```
if (currentlyAllowed) {
  entitlementInfo = blah;
  benefitLevel = blah;
} else  {
  entitlementInfo = /* currently disallowed warning */;
}
```
and below on line 84
```
{entitlementInfo}
{benefitLevel}
```
Open to debate though